src/docfx/template/TemplateEngine.cs

```diff
@@ -195,12 +196,18 @@ public string ProcessHtml(ErrorBuilder errors, FilePath file, string html)
         return LocalizationUtility.AddLeftToRightMarker(_cultureInfo, result);
     }
 
-    private string CreateContent(FilePath file, string? mime, JObject pageModel)
+    private string CreateContent(FilePath file, string? mime, ref JObject pageModel)
     {
-        if (JsonSchemaProvider.IsConceptual(mime) || JsonSchemaProvider.IsLandingData(mime))
+        if (JsonSchemaProvider.IsConceptual(mime))
         {
-            // Conceptual and Landing Data
-            return pageModel.Value<string>("conceptual") ?? "";
+            var conceptual = JsonUtility.ToObject<ConceptualModel>(ErrorBuilder.Null, pageModel)!;
```

Comment:
We probably don't need to roundtrip the `ConceptualModel` here, this method can return the transformed HTML directly, something like:
```suggestion
            return AddLinkLocale(pageModel.Value<string>("conceptual") ?? "");
```

Reply:
right